hardware/1Wire/1WireByKernel.cpp

```diff
@@ -241,8 +244,10 @@ void C1WireByKernel::ThreadBuildDevicesList()
 						case _8_channel_addressable_switch:
 						case programmable_resolution_digital_thermometer:
 						case Temperature_memory:
-							m_Devices[device.devid] = new DeviceState(device);
-							m_p1WireBase->Log(LOG_STATUS, "1Wire: Added Device: %s", sLine.c_str());
+							if (m_Devices.count(device.devid) == 0) {
```

Comment:
This is not needed as it is created only at startup
Instead of 'count' I would also advice to use if **(m_Devices.find(device.devid) == m_Devices.end())**
But it's not needed

Reply:
yes, you are right - it's a leftover from my attempt to call this more frequent. 
I will remove that, and thanks for the tip!